frontend/src/components/account/Account.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import EditAccountButton from '../edit-account/EditAccountButton';
+import './Account.css';
+
+
+const Account = () => {
+  const [user, setUser] = useState({});
+  const [token, setToken] = useState(window.localStorage.getItem("token"));
+  const [refresh, setRefresh] = useState(false);
+
+  useEffect(() => {
+    if(token) {
+      fetch("/accounts", {
+        headers: {
+          'Authorization': `Bearer ${token}`
+        }
+      })
+      .then(response => response.json())
+      .then(async data => {
+        window.localStorage.setItem("token", data.token)
+        setToken(window.localStorage.getItem("token"))
+        setUser(data.user)
+      })
+    }
+  }, [refresh])
+
+  const toggleRefresh = () => {
+    // sets whatever is inside refresh to the opposite, triggering a refresh
+    // through useeffect. this is activated whenever the newpostform is
+    // submitted
+    setRefresh(prevRefresh => prevRefresh + 1);
+  };
+
+
+
+  return(
+    <div>
+      <div className="container">
+
+      <div className="flex-container-photo">
+        {user.photo ? (
+          <>
+          <img src={ user.photo } alt="user-profile" />
+          <EditAccountButton toggleRefresh={toggleRefresh} valueToChange={"photo"}/>
+          </>
+        ) : (
+          <>
+          <img src="/happy-fox.jpeg" alt="default-profile" />
+          <EditAccountButton toggleRefresh={toggleRefresh} valueToChange={"photo"}/>
+          </>
+        )}
+      </div>
+
+      <div className="flex-container">
+        <p className="user-name">Username: { user.userName }</p>
+        <EditAccountButton toggleRefresh={toggleRefresh} valueToChange={"userName"}/>
+      </div>
+
+      <div className="flex-container">
+        <p className="user-email">Email address: { user.email }</p>
+        <EditAccountButton toggleRefresh={toggleRefresh} valueToChange={"email"}/>
+      </div>
+
+      <div className="flex-container">
+        <p className="user-password">Password: { user.password }</p>
```

Comment:
not necessarily a requirement but as a nice feature we can display that as hidden characters like this ****** through css styling

Reply:
Nice idea, I'll do this on a new ticket/branch